src/transformers/visitors/toOrchestrationVisitor.ts

```diff
@@ -587,6 +597,13 @@ export default {
           lhs = lhs.baseExpression;
         }
 
+        const name = indicator.isMapping
```

Comment:
Moved name calculation for mappings (`balances[addr]` -> `balances_addr` in node) higher up so whole states also used this naming convention